src/v3/src/transformer/field/transform.ts

```diff
@@ -73,17 +73,6 @@ export const transformStepInputs: StepTransformer = (step?: NextStep): FormBag =
         };
       }
 
-      // populate all input keys to guide auth-js how to proceed in the remediation engine
-      // TODO: logic here can be removed once authjs can handle auto proceed without hints from the downstream
-      const { options: { type } } = uischema as FieldElement;
-      let defaultValue;
-      if (type === 'string') {
-        defaultValue = '';
-      }
-      if (type === 'boolean') {
-        defaultValue = false;
-      }
-      acc.data[name] = defaultValue;
```

Comment:
This fixed the data leftover issue from flow to flow, does that still work with this change?

Reply:
all tests passed